drivers/mfd/mfd_axp192.c

```diff
@@ -50,12 +153,455 @@ static int mfd_axp192_init(const struct device *dev)
 	return 0;
 }
 
+int mfd_axp192_gpio_func_get(const struct device *dev, uint8_t gpio, enum axp192_gpio_func *func)
+{
+	const struct mfd_axp192_config *config = dev->config;
+	int ret;
+	uint8_t reg_fnc;
+
+	if (gpio >= AXP192_GPIO_NUM_PINS) {
+		LOG_ERR("Invalid gpio (%d)", gpio);
+		return -EINVAL;
+	}
+
+	ret = i2c_reg_read_byte_dt(&(config->i2c), gpio_reg_desc[gpio].reg, &reg_fnc);
+	if (ret != 0) {
+		return ret;
+	}
+
+	switch (gpio) {
+	case 0U:
+	case 1U:
```

Comment:
You need to use `__fallthrough` for all fall-through cases.

Reply:
Fixed